stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -3413,16 +3413,16 @@ internal enum _Variant${Self}Storage<${TypeParametersDecl}> : _HashStorageType {
   }
 
   internal mutating func removeAll(keepCapacity keepCapacity: Bool) {
-    if count == 0 {
+    guard count > 0 else {
       return
     }
 
-    if !keepCapacity {
+    guard keepCapacity else {
       self = .Native(NativeStorage.Owner(minimumCapacity: 2))
       return
     }
 
-    if _fastPath(guaranteedNative) {
+    guard !_fastPath(guaranteedNative) else {
```

Comment:
This doesn't make any sense. It's actually harder to read because of the double negative.